Scripts/script-StixCreator.yml

```diff
@@ -0,0 +1,118 @@
+commonfields:
+  id: StixCreator
+  version: -1
+name: StixCreator
+script: |-
+  ''' IMPORTS '''
+  import json
+
+  from stix2 import Indicator, Bundle
+
+  indicators = []
+
+  userArgs = demisto.args().get('indicators','Unknown')
+  if isinstance(userArgs, dict) == True:
+      all_args = json.loads(json.dumps(userArgs))
+  else:
+      all_args = json.loads(demisto.args().get('indicators','Unknown'))
+
+  counter = 0
+  for indicator_fields in all_args:
+      demisto_indicator_type = all_args.get(str(counter)).get('indicator_type','Unknown')
```

Comment:
if you want to get an index in list then do all_args[counter] instead of all_args.get(str(counter))

Reply:
fixed